### PR TITLE
La tonia branch

### PR DESCRIPTION
**Pull request by LaTonia-Mertica**
Good Day @Tyler-Toro,

Three separate pushes from command line. One pull request:
- email id challenge complete w/comments;
- revisited buy laptop challenge for unanswered questions in comments ... all answered, still - would like walk through of example you posted in Slack if we have an opportunity;
- and used align tag instead of style tag to center JTC header in readme ... also did minor overall edits.

Thanks for your time and energy. Talk soon.

### CODE REVIEW

**Comment by LaTonia-Mertica** [comment]
Hello @Tyler-Toro,

Added my completed library challenge with near epic comments and tweaked my email challenge to stop iterative output ... Gaurav helped me with that at office hours ... Have a fabulous weekend.

**Comment by LaTonia-Mertica** [comment]
Howdy @Tyler-Toro,

Feel good about the fine tuning, practice, and setup for completion of my first project. A lot of work. Still, I really enjoy doing this stuff and pray to keep stride as we transition into phase two. Thanks so much for your support to date. Talk soon.

File: La'Tonia/README.md [hunk 1]
```diff
@@ -1,32 +1,44 @@
-# <div style="text-align: center">**JUSTICE THROUGH CODE**</div>
-### *confession by a code master hopeful*
+# <div align="center">**JUSTICE THROUGH CODE**</div>
```
Comment (Tyler-Toro) [inline La'Tonia/README.md:1]: We'll get a lot of practice with this when we get into html and css files - I haven't worked too much with them but this is amazing practice you're getting into!

File: La'Tonia/anagram_practice.py [hunk 2]
```diff
@@ -0,0 +1,76 @@
...
+    for i in range(5, 0, -1):
+        print("\nAttempts Left:", i)
+        guess = input("Guess: ").lower()
+        if guess == word:
+            print("Correct")
+            break
+        if i == 1:
+            print("\nCorrect Answer:", word)
+
+    choice = input("\nContinue? [y/n]: ")
```
Comment (Tyler-Toro) [inline La'Tonia/anagram_practice.py:72]: this is amazing. You can limit user input so only "y" or "n" are allowed

File: La'Tonia/anagram_practice.py [hunk 3]
```diff
@@ -0,0 +1,76 @@
+'''
+practice project courtesy favtutor:
+https://favtutor.com/blog-details/7-Python-Projects-For-Beginners
+'''
+
+# import not working and cannot run the code but ideally below code adds json and package to provide random/shuffled words ... code on lines 8 to 11 no good
+
```
Comment (Tyler-Toro) [inline La'Tonia/anagram_practice.py:7]: I want to see this code run  - maybe during my office hours this week?

File: La'Tonia/dice_roll_practice.py [hunk 4]
```diff
@@ -0,0 +1,55 @@
+'''
+practice project courtesy favtutor:
+https://favtutor.com/blog-details/7-Python-Projects-For-Beginners
+'''
+
```
Comment (Tyler-Toro) [inline La'Tonia/dice_roll_practice.py:5]: These examples are dope - can you post links in the group chat?

File: La'Tonia/email_ids_challenge.py [hunk 5]
```diff
@@ -0,0 +1,72 @@
...
+    names_list = lower_name.split(' ')
+    joined_names = '.'.join(names_list)
+    email = joined_names + '@ripplemedia.com'
+    emails.append(email)
+
+pprint(emails)
+print()     
+
+# print(lower_name) ... to check what code is doing at this stage
+# print(names_list) ... to check what code is doing at this stage
```
Comment (Tyler-Toro) [inline La'Tonia/email_ids_challenge.py:67]: Good use of sanity checks

File: La'Tonia/guess_game_practice.py [hunk 6]
```diff
@@ -0,0 +1,30 @@
+# practice project courtesy favtutor:
+# https://favtutor.com/blog-details/7-Python-Projects-For-Beginners
+
+# adds random package to the code to generate unknown numbers
+import random
+# creates variable named number assigned to random package with random integer method containing parameters for start and end numbers
+number = random.randint(1, 10)
```
Comment (Tyler-Toro) [inline La'Tonia/guess_game_practice.py:7]: some of these puzzles remind me of my program. I should pull some out for examples

File: La'Tonia/hangman_practice.py [hunk 7]
```diff
@@ -0,0 +1,128 @@
...
+                    print("     / \    ")
+                if turns == 4:
+                    print("4 turns left")
+                    print(" ---------- ")
+                    print("    \ 0     ")
+                    print("      |     ")
+                    print("     / \    ")
+                if turns == 3:
+                    print("3 turns left")
+                    print(" ---------- ")
```
Comment (Tyler-Toro) [inline La'Tonia/hangman_practice.py:91]: These stick figures had me laughing

File: La'Tonia/interactive_dictionary_practice.py [hunk 8]
```diff
@@ -0,0 +1,54 @@
...
+        return data[word]
+# looks for word in data with word title method then returns word title on line 22
+    elif word.title() in data:
+        return data[word.title()]
+# changes to uppercase letters
+    elif word.upper() in data:
+# looks for word in data targeting the word with upper method ?
+        return data[word.upper()]
+# unsuure what is happening here - see length function . ..
+    elif len(get_close_matches(word, data.keys())) > 0:
```
Comment (Tyler-Toro) [inline La'Tonia/interactive_dictionary_practice.py:28]: I'll need to spend more time on this one too!

File: La'Tonia/library_challenge.py [hunk 9]
```diff
@@ -0,0 +1,254 @@
...
+print()
+
+
+def search_by_author(str):
+    author_titles = []
+    for i in books_with_details:
+        if str == i['author']:
+            author_titles.append(i['title'])
+            #print(title)
+        else:
```
Comment (Tyler-Toro) [inline La'Tonia/library_challenge.py:244]: Good use of else, helpful to have, so you wont have an error code if someone enters wrong spelling or something

File: La'Tonia/rock_paper_scissors.py [hunk 10]
```diff
@@ -0,0 +1,45 @@
...
+# GENERAL NOTES: Type in terminal/on command line control letter c to stop game 
+
+# imports random integer from random library
+from random import randint
+
+# assign choice_options variable to list of options user can pick
+choice_options = ["rock", "paper", "scissors"]
+
+# assigns a random play to the opponent of the user/opponent is artificial_intelli_simul aka the computer
+artificial_intelli_simul = choice_options[randint(0,2)]
```
Comment (Tyler-Toro) [inline La'Tonia/rock_paper_scissors.py:14]: I would have name function "AI_sim " such a long function name! lol

File: La'Tonia/tic_tac_toe.py [hunk 11]
```diff
@@ -0,0 +1,138 @@
+'''
+practice project courtesy favtutor:
+https://favtutor.com/blog-details/7-Python-Projects-For-Beginners
+'''
+
+# this script contains multiple new elements to me such as .count() and other methods/functions - which include built-ins and imported ... is solid practice overall and enjoyed learning a bit more design
```
Comment (Tyler-Toro) [inline La'Tonia/tic_tac_toe.py:6]: I wonder if you could transform this into battleship?

**Review by Tyler-Toro — APPROVED** [review]
I come back to 11 files! I really like the outside challenges, those should be included for all JTC

Great work with emails & laptops

any ideas for repo readme?

**Comment by LaTonia-Mertica** [comment]
Thanks - tiring but definitely worth the effort.

If you can help me figure out the two projects I wasn't able to get
working, that'd be awesome. Appears they just need working links (i.e. to
.json files). I emailed the site owner in hopes they'll help,too.

Think I like the y/n option better. Is more streamlined. Still, I'm still
learning. And, lol ... didn't even occur to me to just use AI_sim - rookie
mistake. Building the stick figures and tic tac toe was so much work -
tedious work. But so worth the experience. I try to always do my own coding
versus copy/paste so the coding pangs will definitely stick with me. I'll
probably try to find a battleship version to try it.

I'd love to share the challenges in group chat but it may be better
received if presented neutrally (i.e you post it and encourage folks to try
them). I like these kinds of challenges because they are good for quick
project-based learning ... muscle memory, syntax, etc., and bonus is when
they mirror our curriculum. And bonus bonus you have working code when done
to remind yourself that you can do this. You should definitely pull some of
your code out to share with us. It helps me at least to
visualize possibilities and energizes me to want to progress seeing others'
work.

Anyhoot, thanks for all the feedback. Talk soon.


*___________________________________________*
*La'Tonia Mertica*
(Luh Tone Yuh Merr Treece)
*Founder & Executive Officer*

La'Tonia Mertica 'Communications & Creatives Specialist'
<https://mailchi.mp/50361129f72d/latoniamertica>

2200 South Salina Street #684
Syracuse, New York 13205
***@***.***

**Pull request merged**